.yapfignore

```diff
@@ -0,0 +1,8 @@
+# TODO
+pl_examples/*
+
+# TODO
+pytorch_lightning/*
+
+# TODO
+tests/*
```

Comment:
```suggestion
.git/*

# TODO
pl_examples/*

# TODO
pytorch_lightning/*

# TODO
tests/*
```
it seems this is needed on windows, without it running yapf on root folder crashes on python files in .git folder